output/json.go

```diff
@@ -0,0 +1,75 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
+)
+
+var StatusNames = map[execute.Status]string{
+	execute.Success: "success",
+	execute.Failed:  "failed",
+	execute.Skipped: "skipped",
+}
+
+type JSONTestReport struct {
```

Comment:
Although, if the usage of this package doesn't involve actually accessing these types, you should make them private (`testReport` or `jsonTestReport`).